src/usecases/sendBookingNotification.test.js

```diff
@@ -29,6 +32,27 @@ describe("sendBookingNotification", () => {
       getSendTextMessage: () => sendTextMessage,
       getSendEmail: () => sendEmail,
     };
+
+    process.env.SMS_INITIAL_TEMPLATE_ID = "1";
+    process.env.SMS_UPDATED_VISIT_TEMPLATE_ID = "2";
+    process.env.SMS_JOIN_TEMPLATE_ID = "3";
+    process.env.EMAIL_INITIAL_TEMPLATE_ID = "4";
+    process.env.EMAIL_UPDATED_VISIT_TEMPLATE_ID = "5";
+    process.env.EMAIL_JOIN_TEMPLATE_ID = "6";
+
+    textMessageTemplateId = TemplateStore().firstText.templateId;
+    emailTemplateId = TemplateStore().firstEmail.templateId;
+    textMessageUpdatedTemplateId = TemplateStore().updatedVisitText.templateId;
+    emailUpdatedTemplateId = TemplateStore().updatedVisitEmail.templateId;
+  });
+
+  afterEach(() => {
+    process.env.SMS_INITIAL_TEMPLATE_ID = "";
+    process.env.SMS_UPDATED_VISIT_TEMPLATE_ID = "";
+    process.env.SMS_JOIN_TEMPLATE_ID = "";
+    process.env.EMAIL_INITIAL_TEMPLATE_ID = "";
+    process.env.EMAIL_UPDATED_VISIT_TEMPLATE_ID = "";
+    process.env.EMAIL_JOIN_TEMPLATE_ID = "";
```

Comment:
If the `TemplateStore` is being injected now, would it not be possible to mock it out so we don't need to reset the env vars between each test?

Reply:
I decided not to go down the route of injecting the template store via the app container in the end and went for simpler implementation (just changing it to a function and setting env variables in the test)